src/molecules/TextWithIcon/index.js

```diff
@@ -23,6 +23,7 @@ TextWithIcon.displayName = 'TextWithIcon';
 TextWithIcon.defaultProps = {
   position: 'absolute',
   backgroundSize: 'cover',
+  color: 'white',
```

Comment:
@prasanna1211 why do we need `color` property for span?

Reply:
You can do it like this, if you need color for the text.
<TextWithIcon>
  <Text.p color='white'>Test</Text.p>
</TextWithIcon>